components/suggest/src/store.rs

```diff
@@ -331,6 +334,33 @@ where
                         },
                     )?;
                 }
+                SuggestRecord::Yelp => {
+                    let Some(attachment) = record.attachment.as_ref() else {
```

Comment:
There is a helper function [`ingest_suggestion_from_record()`](https://github.com/mozilla/application-services/blob/main/components/suggest/src/store.rs#L339) to use to save some boilerplate code, can you try using that here? You can find examples in the `AMO` and `Pocket` branches above.

Reply:
The reason why I did so was the data of Yelp suggestion is not an array, unlike other suggestions.
But, yes, I will use `ingest_suggestion_from_record()`, as it looks better to keep the deserialization consistency. Thanks!